### PR TITLE
Javascript light bulb

### PR DESCRIPTION
**Pull request by Hagop420**
Image 11-7-23 at 10 40 AM

### CODE REVIEW

**Review by shawnkost — CHANGES_REQUESTED** [review]
Please include a recording of you toggling back and forth so I can verify this works

File: javascript-light-bulb/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        } else if (mainBulb.className === 'Main_Bulb grey') {
+          document.body.style.backgroundColor = '#fff';
+
+          mainBulb.className = 'Main_Bulb';
+          mainBulb.style.boxShadow = '6px 12px 1rem black';
+        }
+      });
+
+      // edits
+    </script>
```
Comment (shawnkost) [inline javascript-light-bulb/index.html:33]: This should be in a separate JS file

File: javascript-light-bulb/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./bulb.css" />
+    <title>Light Bulb Switcher</title>
+  </head>
+  <body>
+    <div id="flexing-bulb">
+      <div class="Main_Bulb"></div>
```
Comment (shawnkost) [inline javascript-light-bulb/index.html:11]: classes should either be `camelCased` or `kebab-cased`. Classes shouldn't start with a capital letter

File: javascript-light-bulb/index.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        if (mainBulb.className === 'Main_Bulb') {
+          document.body.style.backgroundColor = '#000';
+          mainBulb.className = 'Main_Bulb grey';
+          mainBulb.style.boxShadow = '2px 2px 1rem #f4f4f4';
+        } else if (mainBulb.className === 'Main_Bulb grey') {
+          document.body.style.backgroundColor = '#fff';
+
+          mainBulb.className = 'Main_Bulb';
+          mainBulb.style.boxShadow = '6px 12px 1rem black';
+        }
```
Comment (shawnkost) [inline javascript-light-bulb/index.html:29]: No inline styling. You want to avoid inline styling. You should be adding or removing classes

**Comment by Hagop420** [comment]
https://github.com/Hagop420/c1023-code-solutions/assets/81959688/019814f4-bbf6-4ba4-9b2b-b1d7a70038b0

File: javascript-light-bulb/index.html (hunk 3 above)
Comment (Hagop420) [reply javascript-light-bulb/index.html:29]: Recording 2023-11-07 at 11 19 38 (1)

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**